activity_calendar/templates/activity_calendar/slot_blocks/slot_block.html

```diff
@@ -15,7 +15,15 @@ <h5 class="card-title">
                 {% if slot.location %}
                     <div class="row no-gutters">
                         <small class="text-muted">
-                            Alternative Location: {{ slot.location }}
+                            Alternative Location: {{ sign_up }}
```

Comment:
I can't find where {{sign_up}} is set, nor see why this should be here. Have you accidentally left this here?